frontend/src/views/CellGuide/components/common/OntologyDagView/components/AnimatedNodes/index.tsx

```diff
@@ -209,3 +232,13 @@ function collapseAllDescendants(node: HierarchyPointNode<TreeNodeWithState>) {
     }
   }
 }
+
+function findSVGParent(node: Node | null): SVGElement | undefined {
```

Comment:
this finds the first parent that is an SVG element to be able to calculate coordinates for the tooltip using "localPoint".